owncloud/test/test.py

```diff
@@ -782,6 +782,37 @@ def test_delete_attribute(self, attr1, value1):
         self.assertIsNone(self.client.get_attribute(self.app_name, attr1))
         self.assertEquals(self.client.get_attribute(self.app_name), [])
 
+class TestUserAndGroupActions(unittest.TestCase):
+
+    def setUp(self):
+        self.client = owncloud.Client(Config['owncloud_url'], single_session = Config['single_session'])
```

Comment:
call `self.client.get_config()`. If OC version is < 7, skip this test suite.
There is no provisioning API in OC 6.